src/Plugin/TfaSetup/GALoginHotpSetup.php

```diff
@@ -168,23 +167,13 @@ protected function createSeed() {
   }
 
   /**
-   * Save seed for account.
+   * Setter for OTP secret key.
    *
    * @param string $seed
-   *   Un-encrypted seed.
+   *   The OTP secret key.
    */
-  protected function storeSeed($seed) {
-    // Encrypt seed for storage.
```

Comment:
This has been shifted to the validation class as setup extends that class so this is more accessible now.